cves/kernel/CVE-2016-8650.yml

```diff
@@ -147,10 +150,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    It is not explained how the vulnerability was found, I looked through
+    openwall, bugzilla, redhat, and all the links on mitre
+  automated: null
+  contest: null
+  developer: null
```

Comment:
You can make these false instead of null - that's probably why the CI is failing.